include/mbedtls/ssl.h

```diff
@@ -1165,8 +1164,8 @@ struct mbedtls_ssl_session
 typedef enum
 {
     MBEDTLS_SSL_VERSION_UNKNOWN, /*!< Context not in use or version not yet negotiated. */
-    MBEDTLS_SSL_VERSION_1_2,     /*!< (D)TLS 1.2 */
-    MBEDTLS_SSL_VERSION_1_3,     /*!< (D)TLS 1.3 */
+    MBEDTLS_SSL_TLS1_2 = 0x0303, /*!< (D)TLS 1.2 */
```

Comment:
I can make the change, but would prefer not to have to do it again and again.  Can we agree on `MBEDTLS_SSL_VERSION_TLS1_2/3`?

Reply:
Thanks, that's ok with me and I think well aligned with our various current practices thus I am rather confident that this should be the final word. Otherwise we (Mbed TLS team) will handle that in a subsequent PR.

Reply:
:rocket: